storage/watchable_store_bench_test.go

```diff
@@ -0,0 +1,67 @@
+// Copyright 2015 CoreOS, Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package storage
+
+import (
+	"math/rand"
+	"os"
+	"testing"
+)
+
+// This benchmarks the cancel function for unsynced member
+// in a WatchableStore.
+func BenchmarkWatchableStoreUnsyncedCancel(b *testing.B) {
+	benchSampleSize := b.N
+	// create 2*N watchers and will cancel N watchers at the end
```

Comment:
Ok I will in the next commit. Thanks,

(this branch needs to be rebased, I will do it in the next push)